### PR TITLE
grapple 0.3.1 (new formula)

### PR DESCRIPTION
**Pull request by bric3**
This formula adds a great download accelerator written in rust.
More on the repo readme https://github.com/daveallie/grapple

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by issyl0 — CHANGES_REQUESTED** [review]

File: Formula/grapple.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "eda7ca1bc01ee42e63d2ea3d55f4d6ec0d7d66fd6825aad1481d08a38cb64ae7"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+  end
+
+  test do
+    output = shell_output("#{bin}/kubeseal --version")
```
[the remark this answers is not in the document]
Comment (bric3) [reply Formula/grapple.rb]: Hi, thanks I missed the kubeseal copy/paste.
Also thanks for the test tips !

[the remark this answers is not in the document]
Comment (bric3) [reply Formula/grapple.rb]: Done
I used the URL that was used as an exmple there : https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests

**Comment by bric3** [comment]
Can anyone chime in on this one?

**Comment by iMichka** [comment]
> GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)

This looks like a low-usage tool with not much visibility. See https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

Sorry but we can't accept this formula right now.

**Comment by bric3** [comment]
@iMichka Too bad, but I understand. Thanks for looking at the PR !

**Pull request closed without merging**